static/src/stylesheets/module/_adslot.scss

```diff
@@ -161,6 +154,13 @@
         padding-bottom: $gs-row-height / 2;
         padding-top: $gs-row-height / 2;
 
+        /* Oh horrible code, I'm looking at you. One day, you'll disappear.
+           One day, I'll fucking rip you apart.
+         */
+        &.ad-slot--h250 {
+            height: 250px;
```

Comment:
That's a very good question. As the intent is to eventually remove the top slot animation, I would say yes.